framer/Animation.coffee

```diff
@@ -280,21 +279,7 @@ class exports.Animation extends BaseClass
 	_prepareUpdateValues: =>
 		@_valueUpdaters = {}
 		for k, v of @_stateB
-			if SVGPath.isPath(v)
```

Comment:
What does removing this break?

Reply:
Ah yes, that is part that is not done yet. It breaks animation along a path